friends/src/App.js

```diff
@@ -1,21 +1,29 @@
 import React, { Component } from 'react';
```

Comment:
Your code is good 👌 , but just in case there are gaps like the ones I see in the commit history, just format your code in the meantime